Assets/Scripts/RL/TeamGroup.cs

```diff
@@ -43,6 +45,16 @@ public void ResetTeam(int teamId)
             storage.RemoveAllItems();
```

Comment:
Not your problem, but can you reset the storage to be active in case it was destroyed?

Reply:
Will do